app/src/main/java/org/openobservatory/ooniprobe/common/ResubmitTask.java

```diff
@@ -35,18 +35,19 @@
      */
     public ResubmitTask(A activity) {
         super(activity, true, false);
+        // FIXME: we should use MKResources.
```

Comment:
I've created https://github.com/ooni/probe-android/issues/252 to remember us that we should be using MKResources globally instead of using the previous approach.